analysis/spectra_chisq/_synthetic_photometry.py

```diff
@@ -261,3 +264,51 @@ def plot_synthetic_results(observed, synthetic):
     axis.invert_yaxis()
     axis.legend()
     return fig, axis
+
+
+def get_synth_photo_for_id(obj_id, spec_release, phot_release, time, err_ratio=0.03, temp_flux=None, temp_spec_tbl=None, out_path=None):
+    """Get synthetic magnitudes for one object id using the flux from sncosmo model with extinction.
+    This function is used with ``_model_spectra`` module to fit extinction parameters using synthetic photometry and
+    observed photometry.
+
+    Args:
+
+    Returns:
+    """
+
+    ''' Do the following once: get photo_mag (well it will be in the final table anyway...)'''
+
+
+    if temp_flux is None:
+        # Get synthetic photometry for object id
+        spec_data = spec_release.get_data_for_id(obj_id).copy()
+        #spec_data = unique(spec_data, keys='time')
+
+    # TODO only one band is used in temp_flux.... change this. Want one fit for all bands?
+    if temp_flux is not None and temp_spec_tbl is not None:
```

Comment:
What are you trying to do here? I'm not clear why `temp_flux` and `temp_spec_tbl` are necessary?